### ISSUE TITLE
User creation fails

### ISSUE THREAD

**Issue by dominformant** [Webreaper/Damselfly#457]
When I try to add a new user, Damselfly fails with the following error:
```
System.NullReferenceException: Arg_NullReferenceException
   at Damselfly.Web.Client.Shared.UserManagement.OpenUserDialog(AppIdentityUser user)
   at Microsoft.AspNetCore.Components.ComponentBase.CallStateHasChangedOnAsyncCompletion(Task task)
   at Microsoft.AspNetCore.Components.RenderTree.Renderer.GetErrorHandledTask(Task , ComponentState )
```

**Comment by Webreaper** [comment]
Is this from the user management tab? Or when the popup happens in the config screen? And is this when adding the first user?

**Comment by dominformant** [comment]
This happens, when I click on "Add New User" in the User Management Tab in the settings. And I'm currently trying to add a second user, as the first already existed from a previous version.

**Comment by Webreaper** [comment]
Got it. Fix incoming. :)

**Comment by dominformant** [comment]
Thank you so much for creating and supporting Damselfly

**Comment by Webreaper** [comment]
My pleasure! Fixed in 4.0.3.
